grouper/fe/settings.py

```diff
@@ -25,6 +25,7 @@ def override_timezone(self, value):
     "permission_request_text_help": None,
     "port": 8989,
     "refresh_interval": 60,
+    "shell": [["/bin/bash", "bash"], ["/bin/zsh", "zsh"]],
```

Comment:
Originally it was, but I had to change this so we could have these set for the tests accompanying this change. I tried to patch the fe.settings object for the test, but the interaction between mock.patch and pytest.mark.gen_test led to that not working. If you know of another way of working around that issue, please let me know :)